docs/spdx_output_golang_1.12_alpine.txt

```diff
@@ -0,0 +1,272 @@
+SPDXVersion: SPDX-2.1
+DataLicense: CC0-1.0
+SPDXID: SPDXRef-DOCUMENT
+DocumentName: Tern report for c733097984-golang-1.12-alpine
+DocumentNamespace: https://spdx.org/spdxdocs/tern-report-c733097984-golang-1.12-alpine-e1623d500a67a02164dc5b22eadb4d2d2c0e738d
+LicenseListVersion: 3.4
+Creator: Tool: tern-e1623d500a67a02164dc5b22eadb4d2d2c0e738d
+Created: 2019-05-21T20:41:03Z
+DocumentComment: <text>This document was generated by the Tern Project: https://github.com/vmware/tern</text>
+
+PackageName: golang
+PackageVersion: 1.12-alpine
+PackageDownloadLocation: golang:1.12-alpine
+SPDXID: SPDXRef-c733097984-golang-1.12-alpine
+PackageLicenseDeclared: NOASSERTION
+PackageLicenseConcluded: NOASSERTION
+PackageCopyrightText: NOASSERTION
+FilesAnalyzed: false
+PackageComment: <text>Docker image: golang:1.12-alpine:
+</text>
+
+Relationship: SPDXRef-c733097984-golang-1.12-alpine CONTAINS SPDXRef-f1b5933fe4
+Relationship: SPDXRef-c733097984-golang-1.12-alpine CONTAINS SPDXRef-b87598efb2
+Relationship: SPDXRef-c733097984-golang-1.12-alpine CONTAINS SPDXRef-968d46c1d2
+Relationship: SPDXRef-c733097984-golang-1.12-alpine CONTAINS SPDXRef-1f6ba46a9c
+Relationship: SPDXRef-c733097984-golang-1.12-alpine CONTAINS SPDXRef-23419e1868
+
+PackageName: f1b5933fe4b5f49bbe8258745cf396afe07e625bdab3168e364daf7c956b6b81
+PackageChecksum: SHA256: 820b5a21e229f23001048d25d04ecc8af9210e6983ff529513aad75ead5416ce
+SPDXID: SPDXRef-f1b5933fe4
+PackageDownloadLocation: golang:1.12-alpine
+PackageLicenseDeclared: NOASSERTION
+PackageLicenseConcluded: NOASSERTION
+PackageCopyrightText: NOASSERTION
+FilesAnalyzed: false
+PackageComment: <text>Layer: 820b5a21e2:
+	info: Loading packages from cache for layer 820b5a21e2
+</text>
+
+Relationship: SPDXRef-f1b5933fe4 CONTAINS SPDXRef-musl.musl-1.1.20-r4
```

Comment:
I get errors like this:
`Relationship at line 40 invalid: Missing required license text for LicenseRef-MIT in musl`

That means there should be some reference to some text in the document itself?